src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4024,42 +3960,39 @@ private AccessorListSyntax ParseAccessorList(bool isEvent)
             {
                 // parse property accessors
                 var builder = _pool.Allocate<AccessorDeclarationSyntax>();
-                try
```

Comment:
above we do the pattern of `default(SyntaxList<AccessorDeclarationSyntax>)` and then lazily creating the builder.  i'm super sketical this is worth it.  but i didn't want to change that as i didn't want to accidentally cause a regression.  I'll do a perf analysis of such a change in a spearate change in the future.